app/views/trucks/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>All Trucks</h1>
+
```

Comment:
You might want to wrap this in an:
```
if @trucks
  .. all your logic
end
```

incase there aren't any truck records in your production or development database.